src/i18n/pt-BR/index.js

```diff
@@ -6,85 +6,85 @@ export default {
     message: {
       ...potato,
       help: {
-        label: 'Summon Wizard'
+        label: 'Sumonar mago'
       },
       print: { ...print },
       tester: { ...tester },
       keyboard: {
-        label: 'keyboard'
+        label: 'telcado'
       },
       layout: {
         label: 'layout'
       },
       keymapName: {
-        label: 'Keymap Name',
-        placeholder: 'custom keymap name'
+        label: 'Nome da keymap',
+        placeholder: 'nome customizavel da keymap'
```

Comment:
It isn’t the name that is customized, but rather the keymap so I’d use “nome da keymap customizada”. What do you think?

Reply:
'keymap' também me faz mais sentido como um subjetivo masculino